core/azure-core-auth/src/main/java/com/azure/core/auth/credentials/AzureTokenCredentials.java

```diff
@@ -17,7 +17,7 @@
  * AzureTokenCredentials represents a credentials object with access to Azure
```

Comment:
azure-core-auth will basically have duplicate functionalities as the identity library. Should we remove? @JonathanGiles

Reply:
What's in both packages?
- client secret credential
- client certificate credential
- MSI credential

What's in azure-core-auth, but not in azure-identity?
- Azure CLI credential (use the credential logged in in Azure CLI, would love to have this in azure-identity too)
- User credential (I think Scott is planning to add after preview)
- Auth file (a java properties file with fields similar to the environment credential in azure-identity, used only for management libraries today)

What's in azure-identity but not azure-core-auth?
- ChainedCredential
- AzureCredential (the default chain)
- EnvironmentCredential

Reply:
I think that these types should be removed from azure-core-auth.  Accross all languages we're only exposing the TokenCredential abstraction and the BearerTokenAuthenticationPolicy.  Because these types will have dependencies on ADAL or eventually MSAL they shouldn't be in a core library.